qiita_db/data.py

```diff
@@ -498,11 +512,11 @@ def remove_filepath(self, fp):
             raise e
 
         # We can already update the status to done, as the files have been
-        # unlinked, the purge_filepaths call will not change the status
-        # of the raw data object
+        # unlinked, the move_filepaths_to_upload_folder call will not change
+        # the status of the raw data object
         self._set_link_filepaths_status("idle")
 
-        # Delete the file if it is not used anywhere
+        # Delete the files, if they are not used anywhere
         purge_filepaths(conn_handler)
```

Comment:
We should change that at some point, since the code is not used anywhere, I think that adding an issue to keep track of this will be enough